src/pages/chat.tsx

```diff
@@ -23,32 +54,44 @@ const ChatPage: NextPage = (props): JSX.Element => {
                             <div className="flex flex-col leading-tight">
                                 <div className="text-2xl mt-1 flex items-center">
                                     <span className="text-gray-700 mr-3">
-                                        StinkyGiraffe#7468
+                                        {session.user?.email?.split('@')[0]}
                                     </span>
                                 </div>
                             </div>
                         </div>
                         <div className="flex items-center space-x-2">
-                            {/* Menu Button */}
+                            {/* Sign Out Button */}
                             <button
+                                onClick={() => {
+                                    signOut({
+                                        callbackUrl: `${window.location.origin}`,
+                                    });
+                                }}
+                                className="px-4 py-1 mr-3 text-white rounded bg-dominant hover:bg-accent"
+                            >
+                                Sign Out
+                            </button>
+
+                            {/* Menu Button */}
+                            {/* <button
```

Comment:
Is this commented because it is yet to be implemented? If so, can you or Aastha open the ticket so that future reviewers will understand why this part is commented out?

Reply:
We removed it for the moment, we can add it later if we need it.